### ISSUE TITLE
Live Logs View Show Player Affecting Events like do_punish

### ISSUE THREAD

**Issue by jordantrizz** [MarechJ/hll_rcon_tool#496]
Since rcon logs don't show admin punish, they're not in the Live->Logs View, but they're in the audit log. These actions should be shown in the Live-Logs View so that admins can be aware of what is occurring on the server versus having to watch two different pages.

Perhaps two boxes on the Live page for audit log and live log, or allowing adding of specific commands in the audit log to show in the live log which can be toggled by checkboxes or filters?

**Comment by LordofAgents** [comment]
In my opinion, this is a problem from T17 side, as they do not log that kind of events atm. The Logs View only shows Logs comming from the game server. Mixing these types up could be a issue.

**Comment by ElGuillermo** [comment]
Agreed. Logs are logs, as sent by the server, we shouldn't add anything to them.

**Comment by jordantrizz** [comment]
I don't host a hll_rcon_tool instance anymore, so you can close this.

The major complaint for admins was not being able to see everything and having to switch between the pages. Do with this as you will.
